src/sql/workbench/services/notebook/common/notebookServiceImpl.ts

```diff
@@ -268,6 +269,15 @@ export class NotebookService extends Disposable implements INotebookService {
 		this._providers.delete(providerId);
 	}
 
+	registerNavigationProvider(provider: INavigationProvider): void {
+		this._navigationProviders.set(provider.providerId, provider);
+	}
+
+	getNavigationProvider(notebookUri: URI): INavigationProvider {
+		let provider = this._navigationProviders.size > 0 ? this._navigationProviders.values().next().value : undefined;
```

Comment:
You never use the URI and also never use the provider ID the provider is registered under. This seems like it should take in a providerId instead?

Reply:
Lucy - given we don't use it, we can remove the notebookUri for now and assume that we just use the 1st nav provider. As we add more in the future, can fix up this method.